pytests/tests/test_rpm_cmdline.py

```diff
@@ -125,7 +125,12 @@ def test_install_as_mixed(utils):
     assert(utils.check_package(pkgname) == True)
     assert(utils.check_package(pkgname2) == True)
 
-
-
-
-
+# test installing a package that has the same name as a file
+# example: touch foo; tdnf install foo
+# (file needs to have "*.rpm" extension to qualify)
+def test_install_same_as_filname(utils):
```

Comment:
add a negative case as well

Reply:
What would a negative test do in this case?

Reply:
A file with some other extension other than rpm

Reply:
That is what this test does exactly. `pkgname` is the name of a package, not a filename ending in `.rpm`.